app/components/primer/alpha/segmented_control/button.html.erb

```diff
@@ -0,0 +1,11 @@
+<li class="SegmentedControl-item" role="listitem" aria-current="<%= @selected %>">
```

Comment:
In PRC, I have `aria-current` on the button not the list item. Should I update my implementation to put it on the list item?

cc @bolonio (this week's a11y first responder)

Reply:
Nevermind @bolonio - this was answered in today's office hours. It should be on the button.